examples/misc_camera_traveling.html

```diff
@@ -0,0 +1,187 @@
+                // Copy the view camera3D to modify its far plane distance.
+                // The will render a CameraHelper with shorter axes.
+                const copyCamera = view.camera.camera3D.clone();
+                copyCamera.far = 50;
+                // Creation of the new mesh :
+                const mesh = new itowns.THREE.CameraHelper(copyCamera);
+                mesh.lookAt(position);
+                mesh.updateMatrixWorld();
+                // Add mesh to scene :
+                pickedPointsGroup.add(mesh);
+                view.notifyChange();
+            }
+
+            // Remove all meshes from scene.
+            function clearMeshes() {
+                pickedPointsGroup.remove(...pickedPointsGroup.children);
+                view.notifyChange();
+            }
+
+            // Array which stores picked camera positions, headings, tilts...
+            let cameraTravel = [];
+            // Camera speed in meters per second
+            let cameraSpeed = 500 / 3.6;
+            // Boolean which states if the camera is performing a travel animation or not
+            let travelOn = 0;
+
+            function saveCurrentCameraTransformOptions() {
+                // Retrieve camera current transform options
+                const cameraTransformOptions = itowns.CameraUtils.getTransformCameraLookingAtTarget(
+                    view,
+                    view.camera.camera3D,
+                );
+                // Save camera current transform options
+                cameraTransformOptions.position = view.camera.camera3D.position.clone();
+                cameraTransformOptions.coord.as(view.referenceCrs, cameraTransformOptions.coord);
+                if (cameraTravel.length > 0) {
+                    // Save the distance between the previous and the current picked points
+                    cameraTransformOptions.distance = cameraTransformOptions.position.distanceTo(
+                        cameraTravel[cameraTravel.length - 1].position,
+                    );
+                    cameraTransformOptions.easing = itowns.CameraUtils.Easing.Linear.None;
+                }
+                // Add the camera transform options to the travel steps list
+                cameraTravel.push(cameraTransformOptions);
+                addMeshToScene(new itowns.Coordinates(view.referenceCrs).setFromVector3(cameraTransformOptions.position))
+            }
+
+            function animateCamera() {
+                const camera = view.camera.camera3D;
+
+                // If travel is off, begin travel
+                if (!travelOn) {
+                    ++travelOn;
+                    // The travel time of each step is computed from the distance ran in the step and from the value of
+                    // cameraSpeed at the time of the travel.
+                    cameraTravel.forEach(function (item, index) {
+                        cameraTravel[index].time = (cameraTravel[index].distance || 0) / cameraSpeed * 1000;
```

Comment:
camera could be computed when the travel is added?

Reply:
It could in deed, but then a change of camera speed in the options menu would not be taken into account for every already picked positions. For example, if I select two positions, and then I change the camera speed from its default `500 km/h` value to a lower value, the actual camera speed when launching the animation will still be `500 km/h`.